### PR TITLE
Small sanity check change to openfoam block

### PR DESCRIPTION
**Pull request by wpoely86**
In openfoam 4.0, the `vorticity` binary is superseded by postproces. I've replaced it by `wdot` which seems to be present in all releases so far.

### CODE REVIEW

**Comment by wpoely86** [comment]
@hajgato FYI

File: easybuild/easyblocks/o/openfoam.py [hunk 1]
```diff
@@ -87,7 +87,8 @@ def extract_step(self):
             try:
                 contents_installdir = os.listdir(self.installdir)
                 # it's one directory but has a wrong name
-                if len(contents_installdir) == 1 and os.path.isdir(os.path.join(self.installdir, contents_installdir[0])):
+                if len(contents_installdir) == 1 and \
+                   os.path.isdir(os.path.join(self.installdir, contents_installdir[0])):
                     source = os.path.join(self.installdir, contents_installdir[0])
```
Comment (boegel) [inline easybuild/easyblocks/o/openfoam.py]: @wpoely86 better move `source` outside of the `if`, I don't like this use of `\`

Comment (wpoely86) [inline easybuild/easyblocks/o/openfoam.py]: if that rocks your boat but there's nothing wrong with backslashes

Comment (boegel) [inline easybuild/easyblocks/o/openfoam.py]: we rarely use them elsewhere in the EB codebase, just a matter of consistency

**Comment by boegel** [comment]
looks ok & tested with a couple of existing OpenFOAM easyconfigs, so good to go, thanks @wpoely86!

**Pull request merged**